info/v2/container.go

```diff
@@ -86,8 +86,11 @@ type ContainerStats struct {
 	// The time of this stat point.
 	Timestamp time.Time `json:"timestamp"`
 	// CPU statistics
-	HasCpu bool        `json:"has_cpu"`
-	Cpu    v1.CpuStats `json:"cpu,omitempty"`
+	HasCpu bool `json:"has_cpu"`
+	// In nanoseconds (aggregated)
+	Cpu v1.CpuStats `json:"cpu,omitempty"`
+	// In nanocores per second (instantaneous)
+	CpuInst v1.CpuStats `json:"cpu_inst,omitempty"`
```

Comment:
Kubelet uses v1 today. If you are not planning to update kubelet to move to v2, you can put it in v1. Else, move it to v2.